src/test/java/ca/bc/gov/iamp/bcparis/message/MessageUtilsEdgeCaseGetValueTest.java

```diff
@@ -26,4 +28,24 @@ public void WithNullStringShouldReturnNullString() {
         Assert.assertNull(result);
     }
 
+    @Test
+    public void Test() {
+        List<String> delimiters = Arrays.asList("SN:", "MT:", "MSID:", "FROM:", "TO:", "SUBJ:", "TEXT:", "RE:",
+                "TestRNS:", // Satellite test
```

Comment:
Can we remove inline comments, consolidate them into a one liner before the delimiters list? that way you've got your list as a one liner.

Reply:
Also not sure about best practices.

Reply:
this is copy paste from https://github.com/bcgov/bcparis-service/blob/4e261104a342c6a2d545ba3b4b196fca06366252/src/main/java/ca/bc/gov/iamp/bcparis/transformation/MessageTransform.java#L60-L68